### ISSUE TITLE
403: Client error - request will be discarded

### ISSUE THREAD

**Issue by aronyi** [ACRA/acra#1288]
While debugging acra 403 error appears. 

        initAcra {
            reportFormat = StringFormat.JSON
            httpSender {
                uri = "http://xxxxx:8089/app/1/report"
                basicAuthLogin = "****"
                basicAuthPassword = "******"
                httpMethod = HttpSender.Method.POST
            }
            sendReportsInDevMode = true
        }

**Version**
 - Android: 14 and 11
 - ACRA: 5.11.3
- Acrarium setup: 
Followed the following guide: 
https://github.com/F43nd1r/Acrarium/wiki/Setup-guide

but needed to use port 8089 instead of 8080.

This issue was already reported some time ago, but no solution was reported..
https://github.com/ACRA/acra/issues/901

**Comment by F43nd1r** [comment]
Question is still the same. Do you see anything in the server logs?

**Comment by aronyi** [comment]
Unfortunately there aren't any error logs, since I have installed version 5.11.3 (it was today past 17 o clock).
As you see the latest error is from 2024-01-18 14:02:58. 
Previously I had installed v.4.11.1, so those errors listed at the bottom are all related to v.4.11.1.

[acrarium_logs.txt](https://github.com/ACRA/acra/files/13980603/acrarium_logs.txt)

**Comment by F43nd1r** [comment]
you need the reporter credentials shown when you create the app (can recreate in admin tab). Users are not for reporting.

**Comment by aronyi** [comment]
yes, of course I tried them in the first place and now again another time, to be sure. Those credentials I meant with the 'auto generated' ones.
After that addtiontionally tried to create specific users for that, just to try if it might work.

**Comment by F43nd1r** [comment]
No, the url should be just `/report`, the app is inferred from the reporter

**Comment by aronyi** [comment]
nice :) with this change it is working!!

**Comment by aronyi** [comment]
To make this thread a bit more value added for other:
So before I tried the acra library 4.11.1
https://mvnrepository.com/artifact/ch.acra/acra

Here I have tried out all possibilities out there.
Setting the formUrl to only '/report'.
& Using the reporter credentials
But I still received that 403 error..

So I migrated from 4.11.1 to 5.11.3
For that I needed to restructure other library dependencies, but it was totally worth it, cause now it is working like charme.
